### ISSUE TITLE
White flash before page loading [Android]

### ISSUE THREAD

**Issue by SeriousHoax** [brave/brave-browser#11185]
## Description 
There's a white flash before a page is loaded with no tabs open in the background. I have dark mode enabled and even force dark mode flag enabled. 

## Steps to reproduce 

   1. Open the browser
   2. Browse to any site but opening a heavy side is better to have a better chance of noticing it.

## Actual result 

White flash/windows before page loading 
<details><summary>Screenshot:</summary>

![1](https://user-images.githubusercontent.com/29574622/89974077-e5a76200-dc83-11ea-90dd-4ee13b897022.gif)

</details><br/>
## Expected result

Dark windows
## Issue reproduces how often 

Every single time
## Version/Channel Information:

Stable
- Can you reproduce this issue with the current Play Store version? 
- Can you reproduce this issue with the current Play Store Beta version? 
- Can you reproduce this issue with the current Play Store Nightly version?

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version: Android 9
   
## Brave version
1.11.105

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by jenabaivab** [comment]
Yes this is very annoying. Chrome and Samsung Internet have already managed to get rid of this. So has Vivaldi upto some extent. I hope Brave can clear these out too

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
